lib/context/auth/auth.context.tsx

```
@@ -1,55 +1,255 @@
+      
+    } catch (err) {
+      const error = err as ApolloError;
+      console.error('Error while fetching user profile:', error);
+      showToast({
+        type: "error",
+        title: t("Profile Error"),
+        message:
+          error.cause?.message || t("An error occurred while fetching profile"),
+      });
+    }
+  }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)